suite-native/atoms/src/Card/Card.tsx

```diff
@@ -36,6 +36,7 @@ const cardInnerContainerStyle = prepareNativeStyle<{
     backgroundColor: utils.colors.backgroundSurfaceElevation1,
     borderRadius: utils.borders.radii.r16,
     padding: noPadding ? 0 : utils.spacings.sp16,
+    ...(noShadow ? utils.boxShadows.none : utils.boxShadows.small),
```

Comment:
```suggestion
        {
            condition: !noShadow,
            style: { ...utils.boxShadows.small },
        },
```

Alternatively, you can leave it within the extend array with condition as it was and spread it there. It should work. But it's just a nit to keep it consistent, either way is fine

Reply:
True, I will put it back to the `extend` array. I will also move the `noPadding` condition there as well to make it consistent.